src/main/java/sirius/biz/jobs/batch/file/ArchiveImportJob.java

```diff
@@ -110,7 +111,7 @@ protected Optional<ExtractedFile> fetchEntry(String fileName) throws IOException
         Enumeration<? extends ZipEntry> zipEntries = zipFile.entries();
         while (zipEntries.hasMoreElements() && taskContext.isActive()) {
             ZipEntry zipEntry = zipEntries.nextElement();
-            if (FILE_FILTER.stream().anyMatch(zipEntry.getName()::startsWith)) {
+            if (Files.isConsideredHidden(zipEntry.getName()) || Files.isConsideredMetadata(zipEntry.getName())) {
```

Comment:
**Hint:** For next time: replacement of the hidden check vs metadata check could have been done in 2 commits.